src/plugin/modules/widgets/similarGenomes.js

```diff
@@ -90,9 +94,13 @@ function (
                     this.container.innerHTML = this.dataLayout(data);
                 })
                 .catch((err) => {
-                    this.container.innerHTML = div({
-                        class: 'alert alert-danger'
-                    }, err.message);
+                    console.log('Error loading similar genomes:', err)
```

Comment:
I found that using a div with the "alert" class wasn't super helpful. It would just turn into a red box with no content. Here I chose to just use the same collapsible box with the error message as plaintext inside. Of course I am open to changes on this.

Reply:
PS. the alert class content should work again if we use `err.data.error` rather than `err.message`, so we can go back to that if we want.

Reply:
It is nice to have the error inside the collapsible, rather than exposing it, honoring the hiding of the widget. Yeah, when, if, we move to typescript this should be easier to deal with because we can wrap api and other errors inside known types. I can't tell you how many times I've implemented a "grok whatever error objects is being returned and create a sensible message from it", always with a big bunch of stuff to handle service errors.